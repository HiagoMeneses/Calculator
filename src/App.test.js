import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders title calculator", () => {
  render(<App />);
  const titleElement = screen.getByText(/Calculadora/i);
  expect(titleElement).toBeInTheDocument();
  expect(titleElement).toMatchSnapshot();
});
