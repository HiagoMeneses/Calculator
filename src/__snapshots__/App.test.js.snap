// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders title calculator 1`] = `
<h1>
  Calculadora
</h1>
`;
